recipes/apr/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "apr" %}
+{% set version = "1.5.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://mirror.metrocast.net/apache/apr/apr-{{ version }}.tar.gz
+  sha256: 1af06e1720a58851d90694a984af18355b65bb0d047be03ec7d659c746d6dbdb
+  patches:
+    # Skip a failing test on Mac due to `poll`'s behavior.
+    #
+    # ref: https://www.mail-archive.com/dev@apr.apache.org/msg26625.html
+    #
+    - macos_testpoll.c.patch
```

Comment:
Needed this patch to workaround Mac's `poll` (mis)behavior, which causes test failures. Please see the ref below or just google. 😒

ref: https://www.mail-archive.com/dev@apr.apache.org/msg26625.html